reference/5.1/Microsoft.PowerShell.Utility/Write-Warning.md

```diff
@@ -108,27 +117,37 @@ Accept wildcard characters: False
 ```
 
 ### CommonParameters
-This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](https://go.microsoft.com/fwlink/?LinkID=113216).
+
+This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable,
+-InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose,
+-WarningAction, and -WarningVariable. For more information, see
+[about_CommonParameters](https://go.microsoft.com/fwlink/?LinkID=113216).
 
 ## INPUTS
 
 ### System.String
-You can pipe a string that contains the warning to **Write-Warning**.
+
+You can pipe a string that contains the warning to `Write-Warning`.
 
 ## OUTPUTS
 
 ### None
-**Write-Warning** writes only to the warning stream.
-It does not generate any other output.
 
-## NOTES
+`Write-Warning` writes only to the warning stream. It does not generate any other output.
 
-* The default value for the $WarningPreference variable is Continue, which displays the warning and then continues executing the command. To determine valid values for a preference variable such as $WarningPreference, set it to a string of random characters, such as "abc". The resulting error message will list the valid values.
+## NOTES
 
-*
+The default value for the `$WarningPreference` variable is `Continue`, which displays the warning
+and then continues executing the command. To determine valid values for a preference variable such
+as `$WarningPreference`, set it to a string of random characters, such as "abc". The resulting error
+message lists the valid values.
 
 ## RELATED LINKS
 
+[about_Output_Streams](/powershell/module/Microsoft.PowerShell.Core/About/about_Output_Streams.md)
+
+[about_Redirection](/powershell/module/Microsoft.PowerShell.Core/About/about_Redirection.md)
```

Comment:
**Site Relative** links shouldn't include the extension.

Reply:
fixed